Leibit.Core/Data/sued/TF.xml

```diff
@@ -0,0 +1,412 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<estw>
+  <station name="Radolfzell" short="RRZ" refNr="56">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="66BP" direction="right" />
+      <block name="56BN2" direction="right" />
+    </track>
+  </station>
+
+  <station name="Schwackenreute" short="RSU" refNr="65">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="65BN" direction="left"/>
+    </track>
+  </station>
+
+  <station name="Eriskirch" short="TEK" refNr="76">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="77BP2" direction="left"/>
+    </track>
+  </station>
+
+  <station name="Baienfurt" short="TBAG" refNr="40">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="40BN2" direction="right"/>
+    </track>
+  </station>
+
+  <station name="Aulendorf" short="TAU" refNr="38">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="38BP69" direction="left"/>
+      <block name="39BP" direction="left"/>
+    </track>
+  </station>
+
+  <station name="Brandbühl" short="RBRB" refNr="66" scheduleFile="g66_____.abf" localOrderFile="bf66____.abf">
+    <track name="52" isPlatform="false">
+      <block name="66G52"/>
+    </track>
+  </station>
+
+  <station name="Stahring" short="RST" refNr="67" scheduleFile="g67_____.abf" localOrderFile="bf67____.abf">
+    <track name="1">
+      <block name="67G1"/>
+      <alternative track="2"/>
+    </track>
+    <track name="2">
+      <block name="67G2"/>
+      <alternative track="1"/>
+    </track>
+  </station>
+
+  <station name="Wahlwies" short="RWW" refNr="67" scheduleFile="wahlwi__.abf">
+    <track name="N1">
+      <block name="67BN1"/>
+    </track>
+  </station>
+
+  <station name="Nenzingen" short="RNN" refNr="65" scheduleFile="nenzin__.abf">
+    <track name="G12">
+      <block name="65G12"/>
```

Comment:
Dieser Block hier passt nicht. Nenzingen befindet sich an der anderen Strecke zwischen Stahringen und Stockach. Damit gibt es das Problem, dass der Block "65BN1" sowohl zu Wahlwies als auch zu Nenzingen gehört. Eine eindeutige Zuordnung bekommt man hier also nicht hin. In solchen Fällen habe ich die betroffenen Haltepunkte meistens gar nicht projektiert, z.B. zwischen Buchholz und Handeloh oder in der Sim Aschaffenburg auf der Strecke Richtung Klingenberg.

Reply:
Den Hp hab ich entfernt.